src/components/AttachmentModal.tsx

```diff
@@ -183,7 +187,7 @@ function AttachmentModal({
     const nope = useSharedValue(false);
     const isOverlayModalVisible = (isReceiptAttachment && isDeleteReceiptConfirmModalVisible) || (!isReceiptAttachment && isAttachmentInvalid);
     const iouType = useMemo(() => (isTrackExpenseAction ? CONST.IOU.TYPE.TRACK : CONST.IOU.TYPE.SUBMIT), [isTrackExpenseAction]);
-
+    const [isAttachmentCarouselScrolling, setIsAttachmentCarouselScrolling] = useState(false);
```

Comment:
Can you please explain what is this state for and why is it required?

Reply:
The explanation is the same as [here](https://github.com/Expensify/App/pull/44807#discussion_r1669923321).